### ISSUE TITLE
Generate snippets for undefined steps

### ISSUE THREAD

**Issue by vitalets** [vitalets/playwright-bdd#17]
Running cucumber with undefined step:
```
> cucumber-js

Failures:

1) Scenario: Check home # features/scenario.feature:3
   ? Given I open url "https://playwright.dev"
       Undefined. Implement with the following snippet:
       
         Given('I open url {string}', async function (string) {
           // Write code here that turns the phrase above into concrete actions
           return 'pending';
         });
```

Running playwright-bdd with undefined step:
```
  1) [chromium] › features/sample.feature.spec.js:6:7 › Playwright site › Check title ──────────────

    Error: Unknown step: I open url "https://playwright.dev"
```

See: https://github.com/cucumber/cucumber-js/blob/main/docs/snippets.md

**Comment by vitalets** [comment]
Now for undefined steps the output is following:
```
1. Missing step definition for "features/scenario.feature:3:7"

Given('I open url {string}', async ({}, arg: string) => {
  // write step definition here...
});
```
Snippet format is automatically detected based on existing definitions style (playwright / cucumber) and (js / ts).

**Comment by vitalets** [comment]
Released in v3.2.0
